ArmPkg/Include/Protocol/ArmPageTableMemoryAllocation.h

```diff
@@ -34,7 +34,7 @@ VOID *
   );
 
 typedef struct _PAGE_TABLE_MEM_ALLOC_PROTOCOL {
-  ALLOCATE_PAGE_TABLE_MEMORY    AllocatePageTableMemory;
+  ALLOCATE_PAGE_TABLE_MEMORY    AllocatePageTableMem;

```

Comment:
Can you please separate the individual changes out into separate commits and REBASE & FF this? One major problem we've had with AARCH64 MSVC changes before is they are all merged together and become unwieldy and hard to deal with.

Reply:
Also, was renaming the field really necessary? I assume the name conflict was not for this but the function itself.

Reply:
Separated logic into 5 commits - let me know what you think